docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,95 @@
+![immutadot logo](https://raw.githubusercontent.com/Zenika/immutadot/master/misc/otter.svg?sanitize=true)
+=================
+
+# Getting started
+
+As you already know immutad●t is a javascript library that help you dealing with nested immutable structure. The easiest way to try immutad●t is to use [runkit](https://npm.runkit.com/immutadot). You can test all the following examples in it. If you rather use immutad●t in local, checkout out the [Installation](https://raw.githubusercontent.com/Zenika/immutadot/master/README.md) section of our README.
```

Comment:
I would start by saying what this page is about : "This guide will give you..."
And just mention runkit like that :
Feel free to try the following examples with [runkit](https://npm.runkit.com/immutadot).

Maybe add a first section "Installation", repeating the main README but it has its place here...